packages/joint-core/src/dia/Graph.mjs

```diff
@@ -396,6 +396,41 @@ export const Graph = Model.extend({
         this.get('cells').remove(cell, { silent: true });
     },
 
+    transferCellEmbeds: function(sourceCell, targetCell, opt = {}) {
+
+        const batchName = 'transfer-embeds';
+        this.startBatch(batchName);
+
+        opt.reparent = true;
```

Comment:
I would not modify the `opt`. It would be better to pass `{ ...opt, reparent: true }` inside the `embed()` function.